<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChatWindow</class>
 <widget class="QMainWindow" name="ChatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="startDate">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(240,240,240);
color : rgb(0,0,0);
border-color: : transparent;</string>
    </property>
    <property name="text">
     <string>2020-07-25</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endDate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color : rgb(240,240,240);
color : rgb(0,0,0);
border-color: : transparent;</string>
    </property>
    <property name="text">
     <string>2020-07-30</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>20</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
                   background-color:rbga(64,66,68,80);;
                   border-style:outset;                                                             
                   border-radius:5px;                     
                   font: 75 15pt &quot;华文行楷&quot;;           
                   padding:6px;
                   }

                   QPushButton:pressed{
                   background-color:rgba(100,100,100,180);
                   border-color:rgba(100,100,100,30);
                   border-style:inset;
                   }
                   QPushButton:hover{
                   background-color:rbga(64,66,68,80);;
                   color:rgba(255,255,255,200);
}
</string>
    </property>
    <property name="text">
     <string>提交</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>801</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>温度</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QtCharts::QChartView" name="temperatureWidget" native="true"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>心率</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QtCharts::QChartView" name="heartRateWidget" native="true"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>血压</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QtCharts::QChartView" name="bloodPresureWidget" native="true"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>血糖</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QtCharts::QChartView" name="bloodGluseWidget" native="true"/>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>血氧</string>
     </attribute>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QtCharts::QChartView" name="bloodOxygenWidget" native="true"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>861</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">backgroud-color: rgb(65,65,65);
color: rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string> 温馨提示：正常情况血糖：3.9-6.1 心率：60-100   动脉血氧约为98% 静脉血氧约为75%   血压：低压60-90，高压95-120</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtCharts::QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
